Makefile

```diff
@@ -38,7 +41,12 @@ ADD_GLYPHS_FLAGS = -a emoji_aliases.txt
 PUA_ADDER = map_pua_emoji.py
 VS_ADDER = add_vs_cmap.py # from nototools
 
-EMOJI_SRC_DIR ?= png/128
+ifdef DOWNLOAD_SOURCE
```

Comment:
how/where is this DOWNLOAD_SOURCE defined?

Reply:
It's defined when passing it to the make command: `make DOWNLOAD="Emoji_test_folder"`